app/controllers/billing.js

```diff
@@ -0,0 +1,6 @@
+import Controller from '@ember/controller';
+import {alias} from '@ember/object/computed';
+
+export default Controller.extend({
+    guid: alias('model')
```

Comment:
I had an eslint error saying I must "alias" my model - so I copied this from controllers/site.js